client/src/components/settings/DeleteModal.tsx

```diff
@@ -36,6 +38,8 @@ function DeleteModal(props) {
         <p>{t('settings.danger.delete-p1')}</p>
         <p>{t('settings.danger.delete-p2')}</p>
         <p>
+          {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+          {/* @ts-ignore */}
           <Trans email={email} i18nKey='settings.danger.delete-p3'>
```

Comment:
```suggestion
          <Trans i18nKey='settings.danger.delete-p3'>
```

Reply:
I see. I haven't used `react-i18next` so wan't too sure what to do there and commented it out for later review. I have the last two files, the test files, remaining to be migrated. Also Oliver had given a link on using TS with `react-i18next`, so have to go through that. I had planned to go through the changes I did once I had migrated all the files incase I missed anything or did something the wrong way.  I'll correct this when I'm doing that review.

Reply:
I've changed this to the one you suggested. But I haven't gone through the link Oliver sent yet. Will do that soon.